stojanmitric/GoogleBooksApiWithGuzzle
# # Searching books by isbn and name with Postman and created uniformed JSON response

#What's the purpose of this MR?

In this pull request is implemented functionality to search through Google Books API by sending post requests (name or isbn of the book) with Postman to App which sends GuzzleHttp get request to Google Books API and return uniformed JSON file containing information of wanted book.

#Impact Analysis

This pull request will possibly affect differences in structure of base lumen app that is added in develop branch, since it has new functionalities added.

#Where should the tester start?

Tester should start from running app locally by starting server on local machine with port for example 50. 

#How should this be manually tested?

Start making post requests using Postman to check if searching books by isbn and name works. 

For example:
Url for isbn post request:

http://localhost:50/GoogleBooksApiWithGuzzle/public/booksIsbn

In postman click on params and enter :

bookIsbn -> as a key 
9781451648546 -> as a value

Url for bookName post request:

http://localhost:50/GoogleBooksApiWithGuzzle/public/booksName

bookName-> as a key 
anything -> as a value

#Any background context you want to provide?

-

#What are the relevant tickets?

Relevant tickets are the one from Trello board.

#Screenshots 
![bookname_post_json](https://user-images.githubusercontent.com/20643073/27121904-5ad0b86c-50e9-11e7-9f1a-e9f214a8228d.JPG)
![isbn_post_json](https://user-images.githubusercontent.com/20643073/27121907-5d67fa9a-50e9-11e7-94c4-e47c590b2803.JPG)
